tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_zip_codes_test.py

```diff
@@ -0,0 +1,67 @@
+"""
+Unit test for generalize_zip_codes module
+
+This cleaning rule generalizes all PPI concepts containing zip codes to their
+primary three digit representation, (e.g. 35400 → 354**).
+
+This cleaning rule is specific to the controlled tier.
+
+
+Original Issue: DC1376
+"""
```

Comment:
Update this description as well with `observation-source_concept_id` 1585250

Reply:
Sounds good, doing these now.